src/jwt.ts

```diff
@@ -4,11 +4,14 @@ import jwtDecode from "jwt-decode";
 
 export interface PublicKey {
   kid: string;
-  alg: string;
+  alg?: string;
   kty: string;
   e: string;
   n: string;
   use: string;
+  x5c?: string[];
+  x5t?: string;
+  issuer?: string;
```

Comment:
Do we need these things in the Public Key or is it just Azure AD Specific.

Reply:
probably don't need the issuer but there is x5t key present in public key, but I guess the test is working without it so I will remove the keys from interface

Reply:
Cool